app/src/main/java/org/wikipedia/navtab/NavTabLayout.kt

```diff
@@ -19,19 +12,5 @@ class NavTabLayout constructor(context: Context, attrs: AttributeSet) : BottomNa
             val navTab = NavTab.of(i)
             menu.add(Menu.NONE, navTab.id(), i, navTab.text()).setIcon(navTab.icon())
         }
-        fixTextStyle()
-    }
-
-    private fun fixTextStyle() {
-        if (isNotEmpty()) {
-            (getChildAt(0) as ViewGroup).forEach {
-                val largeLabel = it.findViewById<TextView>(R.id.largeLabel)
-                largeLabel.ellipsize = TextUtils.TruncateAt.END
-                largeLabel.updatePadding(left = 0, right = 0)
-                val smallLabel = it.findViewById<TextView>(R.id.smallLabel)
-                smallLabel.ellipsize = TextUtils.TruncateAt.END
-                smallLabel.updatePadding(left = 0, right = 0)
-            }
-        }
     }
```

Comment:
They renamed the `ids` and not sure why we cannot use the new `ids`. However, by checking the source code, looks like the settings are default and it should be fine to be removed.